test-harness/modules/mocker/endpoints/mesos.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+"""Mesos mock endpoint"""
+
+import logging
+
+from exceptions import EndpointException
+from mocker.endpoints.recording import (
+    RecordingHTTPRequestHandler,
+    RecordingTcpIpEndpoint,
+)
+
+# pylint: disable=C0103
+log = logging.getLogger(__name__)
+
+
+# pylint: disable=R0903
+class MesosHTTPRequestHandler(RecordingHTTPRequestHandler):
+    """A request hander class mimicking DC/OS IAM (Bouncer) operation.
```

Comment:
> There is a mismatch between the class name and the docstring.

Copy-pasting mistake - both  Bouncer and Mesos endpoints follow the same template.

> Please check all docstrings, or remove them if it's too much of a burden to maintain them.

????

> Most of the class names used here are pretty self-expressive. Docstrings make most sense when they add additional information.

I was following https://google.github.io/styleguide/pyguide.html: "Classes should have a doc string below the class definition describing the class. If your class has public attributes, they should be documented here in an Attributes section and follow the same formatting as a function's Args section. "

In this particular case I think it makes sense - for somebody unfamiliar with the code, it may as well be a handler for requests issues by Mesos.